build.gradle.kts

```diff
@@ -25,6 +25,7 @@ val springDocVersion = "1.7.0"
 val sentryVersion = "7.3.0"
 
 dependencies {
+  implementation("uk.gov.justice.service.hmpps:hmpps-kotlin-spring-boot-starter:1.0.2-beta-4")
```

Comment:
out of interest, can you recall why this was required? I guess maybe to pull in HmppsPrisonProbationSubjectAccessRequestService?

Reply:
yes this is correct, we may need to update to latest, so when I rebase against SB3 i'll pass through to check the version.

Reply:
What this does is pull in the interface and then the service is stood up based on that, automagically.